requirements.txt

```diff
@@ -7,3 +7,4 @@ openpyxl
 openupgradelib
 pygount
 sentry_sdk<=1.9.0
+unidecode
```

Comment:
Put this in the other commit.

Reply:
Done, I forgot this in the other commit.